### ISSUE TITLE
Add virtual mouse and keyboard support for Virtual Display 

### ISSUE THREAD

**Issue by Libvulkan** [Genymobile/scrcpy#5557]
I can't use --keyboard=uhid and --mouse=uhid in virtual Display because it's showing in real Display

**Comment by rom1v** [comment]
Yes, it depends on the device. See discussions around https://github.com/Genymobile/scrcpy/pull/5370#issuecomment-2440181130

**Comment by parkerlreed** [comment]
For two Samsung devices (S20 Plus on Android 13 and Z Flip 3 on 14) it's always going to the internal screen.

Without the pass through I can control the screen but I lose the ability to click if the device goes to sleep (but I can still right click to go back and it responds)

**Comment by rom1v** [comment]
Fixed by #6009, released in scrcpy 3.3.
